tests/dummy/npm-debug.log

```diff
@@ -0,0 +1,47 @@
+0 info it worked if it ends with ok
```

Comment:
Can we exclude this file from the repo?

Reply:
I removed that and added it to gitignore.